util/intstr/parametrizable_test.go

```diff
@@ -1,13 +1,21 @@
 package intstr
 
 import (
+	"encoding/json"
+	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestInt(t *testing.T) {
 	i, err := Int(ParsePtr("2"))
+	fmt.Println("HERE HERE")
```

Comment:
remove Println statements?

Reply:
Yeah, none of this should have been changes.